duckduckgo/iOS
# Update Sync tests

Task/Issue URL: https://app.asana.com/0/1204165176092271/1207389328000082/f
Tech Design URL:
CC:

**Description**:
- Fixes Maestro reporting by switching from Maestro GH Action to CLI.
- Fixes a test that can take too long to execute by splitting it into two parts.



**Steps to test this PR**:
1. Verify CI is green.
2. A successful run of this branch is at https://github.com/duckduckgo/iOS/actions/runs/9192063144




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
